### ISSUE TITLE
Are you still working on this idea?

### ISSUE THREAD

**Issue by vijayee** [chadrickm/buildithub#1]
Are you still working on this idea? I'd like to help contribute after reading your maidsafe post on it. I have been trying to create a similar system.

**Comment by chadrickm** [comment]
At this moment I am involved with other endeavors but I'm still excited about it.

**Comment by vijayee** [comment]
I'd like to pick this idea up. I'm working on a similar project that 
thanks to you I now know is similar to an open value network. I didn't 
know about them till I saw your maidsafe post. I am writing a paper on 
it. Would you be interested in checking it out when I'm done the draft?

**Comment by chadrickm** [comment]
AWESOME! I think it's a great idea and I wasn't unique with it. I've come to find that lots of others are thinking the same stuff. I'd love to read what you come up with.

**Comment by vijayee** [comment]
Also this site is working currently in similar but more focused manner
http://builditwith.me/
 Sent using Hushmail
On 1/27/2015 at 8:07 AM, "Chadrick Mahaffey"  wrote:

```
AWESOME! I think it's a great idea and I wasn't unique with the idea.
```

I've come to find that lots of others are thinking the same stuff. I'd
love to read what you come up with. 

```
—
```

**Comment by chadrickm** [comment]
https://assembly.com/
is another one that seems farther along but I don't believe they have built it with the open value network in mind.

**Comment by mastercyb** [comment]
Interested

**Comment by vijayee** [comment]
@21xhipster In which aspect? I'm currently designing an open value network style system that would incorporate this feature set of collaborative building.

**Comment by chadrickm** [comment]
Also you might want to get hold of [Francis Brunelle](https://www.maidsafe.org/users/frabrunelle/activity) (an admin on maidsafe forum and heavily involved with the dev pods) he is friends with Tiberius Brastaviceanu who has been focused on OVNs for quite some time. See [SENSORICA](http://www.sensorica.co/home/about-us/tiberius-brastaviceanu).

**Comment by chadrickm** [comment]
@vijayee You mentioned you were "...writing a paper on it." I'd love to see it when you're done or before if you want some editorial eyes on it (Google docs is great for this stuff). I've also got a ton of material in a google drive (all my brainstorming ideas and so on) that I'm willing to open up for anyone working toward the goals of an OVN creation/managing framework/toolset.

**Comment by vijayee** [comment]
@chadrickm http://vijayee.gitbooks.io/prometheus-the-social-currency-network/  This is the working draft. I'm in process of moving the paper to Dbook and I'm gathering collaborators.

**Comment by vijayee** [comment]
I also reached out to Francis about Sensorica about a month or so ago. He hadn't heard from them in a while but he said they were frequenting the MaidSafe meetups. There platform seemed to be in python and non-distributed when I looked under the hood.

**Comment by chadrickm** [comment]
Where can we discuss the book/paper? I like your thinking "Precious metals are only useful so much as they can purchase substance that fulfills a basic need. Time however is always valuable no matter context; to every living thing on earth. Furthermore when we fulfill a basic survival needs what do we really gain? More time to live. We in a sense purchase more time with fulfillment of those needs. This makes our needs an abstraction of time." I'm thinking Time = Money as an abstraction is a bit of an oversimplification. If you add Energy and Expertise along with Time (Time (Energy Spent) + Expertise) you have a fuller picture of the abstraction of money. Expertise has to be part of the equation because not everyone's time is "worth" the same amount. Thoughts?

**Comment by vijayee** [comment]
Yes agreed this is a planned part of how goods will be valued. The discussion can happen in https://www.reddit.com/r/prometheus_scn/ or you can email me and I can add you to the slack chat.

**Comment by chadrickm** [comment]
Yeah, add me: chadrickm at gmaily dottie com

**Comment by vijayee** [comment]
You are cordially invited.

**Comment by mastercyb** [comment]
I've read a paper and didn't catch how the network is going to manage different value of sparks? A time of biological agents has a different value by nature.

**Comment by vijayee** [comment]
The value of goods will change but the value of sparks remains the same. The difference in good prices will be handled by negotiations between buyer and seller at the time of sale. The people who put the sparks into circulation still have to provide their original value in time by selling their equivalent value in goods.
